scripts/ama.coffee

```diff
@@ -18,6 +18,7 @@
 # Author:
 #   Gregory Cerna
 #   Steven Diaz
+#   Samuel Tallent
```

Comment:
I changed it to odds, not sure if it's information that's worth being shown every time